src/com/facebook/buck/cli/ProjectCommand.java

```diff
@@ -176,6 +176,12 @@ public static Ide fromString(String string) {
   )
   private boolean skipBuild = false;
 
+  @Option(
+      name = "--enable-extra-compiler-output-modules",
+      usage = "Allow generation of extra modules that provide extra compiler output to IntelliJ."
+  )
+  private boolean enableExtraCompilerOutputModules = false;
```

Comment:
buck config should be enough, i was just following what seemed like convention to do both. i can remove this